crates/sui-framework/packages/move-stdlib/tests/bcs_tests.move

```diff
@@ -101,7 +101,8 @@ module std::bcs_tests {
     }
 
     #[test]
-    #[expected_failure(abort_code = 453, location = std::bcs)]
+    #[expected_failure]
```

Comment:
Why the dropping of the location and code?

Reply:
The error is now that this hits the VM object depth, not the `bcs` decoding error that was previously being thrown.